push-to-cluster.sh

```diff
@@ -0,0 +1,5 @@
+podman build -t quay.io/dkhater/testing:latest .
```

Comment:
I think this script was committed by mistake?

Reply:
sorry not ready for review!